bftengine/include/bcstatetransfer/SimpleBCStateTransfer.hpp

```diff
@@ -167,6 +167,7 @@ struct Config {
   bool runInSeparateThread = false;
   bool enableReservedPages = true;
   bool enableSourceBlocksPreFetch = true;
+  uint32_t getMissingBlocksSummaryWindowSize = 600;
```

Comment:
For the above:
The name I chose a long time ago is not accurate.
Can we change it to gettingMissingBlocksSummaryWindowSize?
nit: consider moving this under // sizes

For this whole file:
To be able to see the parameter printed, add it to operator<<() function below, you missed it.